### DISCUSSION TITLE
github visit error!

### DISCUSSION THREAD

**Discussion by lvzhenbang** [community/community#29739] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/19884132/185010655-265ec8d2-34a6-411d-8895-acd6306333be.png)

**Comment by dannylin0711** [comment]
Z-index not set properly, I think.
The recommended search div gets 35, but recent repository div gets 40.

**Comment by radar-blip** [comment]
Same Issue here: #29743

File a bug report if you want, here: [Bug-Report](https://support.github.com/contact/bug-report)
